config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -260,3 +260,51 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  kubernetes/perf-tests:
+  - name: pull-perf-tests-clusterloader
+    always_run: true
+    skip_report: false
+    max_concurrency: 12
```

Comment:
Given that we're using the same gcp project as the kubernetes/kubernetes presubmit, maybe let's lower that to 2 or so in order to not starve the kubernetes/kubernetes presubmit. We don't have many PRs in kubernetes/perftests, so a small value should be fine